### PR TITLE
DE4830 - Dark Theme Interactive Examples

### PR DESCRIPTION
**Pull request by seancdavis**
The primary focus of this defect was to support dark-theme toggling within interactive examples. I ended up finding a few other dark theme issues and fixed those along the way. All in all, in conjunction with crdschurch/crds-styles#271, this addresses:

- Style support for dark theme on accordions, panels, and striped tables
- Communicating the current theme to interactive examples and toggling to match
- Maintaining a dark theme within interactive examples when navigating to a new page

### CODE REVIEW

File: src/examples/button-dropdown/index.html [hunk 1]
```diff
@@ -1,63 +1,79 @@
...
+    try {
+      Typekit.load({
+        async: true
+      });
+    } catch (e) {}
 
-  <body>
+  </script>
 
-    <style>
```
Comment (zarahzachz) [inline src/examples/button-dropdown/index.html]: Out of curiosity, why move `<style>` tag from `<head>` to `<body>`?

File: src/examples/tooltips/index.html [hunk 2]
```diff
@@ -1,63 +1,79 @@
...
-    <script src="https://unpkg.com/zone.js/dist/zone.js"></script>
-    <script src="https://unpkg.com/zone.js/dist/long-stack-trace-zone.js"></script>
-    <script src="https://unpkg.com/reflect-metadata@0.1.3/Reflect.js"></script>
-    <script src="https://unpkg.com/systemjs@0.19.31/dist/system.js"></script>
-    <script src="config.js"></script>
-    <script>
+
+<head>
+  <base href="." />
+  <title>ng2-bootstrap accordion</title>
```
Comment (zarahzachz) [inline src/examples/tooltips/index.html]: You've changed all previous titles to `NG-X Boostrap [Component]`. Why's it different here?

File: src/examples/video-modals/index.html [hunk 3]
```diff
@@ -1,32 +1,44 @@
 <!DOCTYPE html>
 <html>
-  <head>
-    <base href="." />
-    <title>ng2-bootstrap datepicker</title>
 
-    <link rel="stylesheet" href="/app.css" />
+<head>
+  <base href="." />
+  <title>ng2-bootstrap datepicker</title>
```
Comment (zarahzachz) [inline src/examples/video-modals/index.html]: Should title be updated?

File: src/examples/tooltips/index.html (hunk 2 above)
Comment (seancdavis) [reply src/examples/tooltips/index.html]: Missed that one. Good catch!

File: src/examples/button-dropdown/index.html (hunk 1 above)
Comment (seancdavis) [reply src/examples/button-dropdown/index.html]: It's not moved, the file is just re-indented. I used prettier to re-indent the files. It looks like it replaced content, but it's just being silly.

**Pull request merged**